app/presenters/finder_content_item_presenter.rb

```diff
@@ -50,7 +50,12 @@ def details
       logo_path: file.fetch("logo_path", nil),
       show_summaries: file.fetch("show_summaries", false),
       signup_link: file.fetch("signup_link", nil),
-      summary: file.fetch("summary", nil),
+      summary: [
+        {
+          content_type: "text/govspeak",
+          content: file.fetch("summary", ""),
+        },
+      ],
```

Comment:
Currently, if no summary is provided, we're setting `summary: nil` in the presenter.
With this PR, that would change to:

```ruby
summary: [
  {
    content_type: "text/govspeak",
    content: "",
  }
]
```

I imagine that means we'll end up rendering some sort of `<div class="summary"></div>` empty placeholder on the specialist finder page itself, whereas I imagine the `nil` value might avoid rendering any placeholder.

It's probably best that we continue with that logic (in case the 'summary placeholder' introduces additional padding etc compared to the 'absence of placeholder' we had before).

So I'd suggest something like this instead:

```suggestion
      summary: file.fetch("summary", nil).nil? ? nil : [
        {
          content_type: "text/govspeak",
          content: file.fetch("summary"),
        },
      ],
```